tools/text_processing/text_processing/tool_dependencies.xml

```diff
@@ -1,34 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
Of course it is okay.

xref https://github.com/galaxyproject/galaxy/issues/2446